draft-irtf-nwcrg-coding-and-congestion-04.txt

```diff
@@ -438,35 +493,203 @@ Internet-Draft            Coding and congestion                July 2020
    designed.
 
    Examples of the solution could be adding a given percentage of the
-   congestion window as supplementary packets or sending a given amount
-   of repair packets at a given rate.  The redundancy flow can be
+   congestion window as supplementary symbols or sending a given amount
+   of repair symbols at a given rate.  The redundancy flow can be
    decorrelated from the congestion control that manages source packets:
-   a secondary congestion control could be introduced underneath the FEC
+   a secondary congestion control could be introduced to manage the
```

Comment:
I also stumbled over this, like Emmanuel. First of all, the text that you intend to write would become clearer if you'd change this from "a secondary congestion control" to "a separate congestion control instance".

Second, I think this is not what Emmanuel is concerned about. He wrote "so it means doing CC at an equivalent link-layer? Point-to-point?". I think what you have in mind is end-to-end. Indeed, at a link layer, FEC is happening, and the relationship to CC is that the FEC is a part of the "link" - the total capacity exposed by the link is the link + FEC, I believe. So there needs to be text here that separates this idea (which is e2e) from link layer FEC.

I made a proposal above.

Reply:
I agree with the second point - we are mainly focusing on end-to-end ; tunnels would be looked at in LOOPS